### DISCUSSION TITLE
Some questions regarding schedule expiration deletion, full backup, and cross-cluster recovery

### DISCUSSION THREAD

**Discussion by lizhifengones** [vmware-tanzu/velero#6985] [Community support Q&A]
**env**: minio + openebs(+ nfs) + velero + openes.io/velero-plugin

**My data scenario**: It cannot be deleted or tampered with, so there will be more and more data.

**What I am doing**: Back up pvc, volume snapshot data 

Some questions I have:
1. The first time the schedule is a full backup, will it be an incremental backup in the future? Which parameter should be used to find the previous backup?
2. After the schedule backup expires, it will be cleaned up by velero server gc. Will it not be deleted because it is a reference to the incremental backup?(More and more backup resources are worried about putting pressure on the apiserver)
3. Regardless of whether the backup will be deleted when it expires, can the schedule ignore the last backup and start the backup from 0?(To ensure data integrity)
4. To restore shedule backup across clusters, do you need to create the backups and schcdule resources of the current cluster in advance in the new cluster (for example, kubectl get backup -o yaml copy)

**Comment by draghuram** [comment]
Hi @lizhifengones, I posted in your other question as well. It is better to merge both as answers depend on what I asked there.
